infra/terraform/test-org/org/locals.tf

```diff
@@ -115,6 +115,13 @@ locals {
       description = "Deploys a web-based ecommerce app into a multi-cluster Google Kubernetes Engine setup."
       owners      = ["NimJay"]
     },
+    {
+      name        = "terraform-example-java-dynamic-point-of-sale"
+      short_name  = "java-dynamic-point-of-sale"
+      org         = "GoogleCloudPlatform"
+      description = "Deploys a dynamic Java webapp into a Google Kubernetes Engine cluster."
+      owners      = ["@GoogleCloudPlatform/dee-platform-ops"]
```

Comment:
We don't support groups just yet https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/issues/1505. Could you switch to users for now?

Reply:
Done!